mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -418,6 +418,36 @@ protected StoreMessageManager createMessageManager(Mailbox mailbox, MailboxSessi
     }
 
     @Override
+    public org.apache.james.mailbox.MessageManager getMailbox(MailboxId mailboxId, MailboxSession session)
+            throws MailboxException {
+        MailboxMapper mapper = mailboxSessionMapperFactory.getMailboxMapper(session);
+        Mailbox mailboxRow = mapper.findMailboxById(mailboxId);
+
+        if (mailboxRow == null) {
```

Comment:
true, but it was done like that in getMailbox(MailboxPath) so I was chilly to change it.